packages/paste-core/utilities/text/src/SafelySpreadProps.ts

```diff
@@ -0,0 +1,17 @@
+import {SPACE_PROPS, TYPOGRAPHY_PROPS, OVERFLOW_PROPS} from '@twilio-paste/style-props';
+
+export const PROPS_TO_NULLIFY = {
+  ...SPACE_PROPS,
+  ...TYPOGRAPHY_PROPS,
+  ...OVERFLOW_PROPS,
+  display: undefined,
+  textColor: undefined,
+  verticalAlign: undefined,
+  className: undefined,
+  style: undefined,
```

Comment:
Still need to unset classname and style just in case you're doing "bad things" with our styled components.